src/components/Breadcrumb/Breadcrumb.container.js

```diff
@@ -1,40 +1,44 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
-import { useSelector } from 'react-redux';
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
 
-import propTypes from 'prop-types';
-import Home from '../Home';
+import BreadcrumbComponent from './Breadcrumb.component';
 
-const BreadcrumbContainer = ({ clickableFolders, showingRoute }) => {
-  const darkThemeActive = useSelector((state) => state.player.darkTheme);
+const BreadcrumbContainer = () => {
+  const { path } = useParams();
+
+  const [showingRoute, setShowingRoute] = useState([]);
+  const [clickableFolders, setClickableFolders] = useState([]);
+
+  useEffect(() => {
+    const myRoute = path || '';
+    const cleanRoute = myRoute.replaceAll('%20', ' ');
+
+    const splitBreadCrumb = cleanRoute.split('/');
+    const cleanRoutes = splitBreadCrumb.filter((tempRoute) => tempRoute !== '');
+    const finalRoutes = [];
+
+    for (let i = 0; i < cleanRoutes.length; i += 1) {
+      let tempRoute = '';
+      for (let j = 0; j <= i; j += 1) {
+        tempRoute = `${tempRoute}/${cleanRoutes[j]}`;
+      }
+
+      finalRoutes.push(`${tempRoute}/`);
+    }
+
+    setClickableFolders(finalRoutes);
+    setShowingRoute(cleanRoutes);
```

Comment:
This can be simplified to:

```
const split = path.split("/");

const breadcrumbs = split.reduce((paths, current) => {
  const uri = paths.map(path => path.title);
  return [
    ...paths,
    {
      title: current,
      uri: `${uri.join('/')}/${current}`
    }
  ];
}, []);
```

Having `path = ''` the result is:
```
[{title: "", uri: "/"}]
```

having `path = '/Music/my-favorite-martian'` the result is:
```
[
  {title: "", uri: "/"},
  {title: "Music", uri: "/Music"},
  {title: "my-favorite-martian", uri: "/Music/my-favorite-martian"}
]
```